app/display/editor/src/main/java/org/csstudio/display/builder/editor/DisplayEditor.java

```diff
@@ -19,7 +20,9 @@
 import java.util.logging.Level;
 import java.util.prefs.Preferences;
 
+import javafx.scene.control.*;

```

Comment:
We generally use individual imports, don't let the IDE collapse them into wildcard imports

Reply:
Yes, I did not see my IDE collapse that, I did the correction, I'm just waiting for the middle click response and I'll re-push.